src/viem/saga.ts

```diff
@@ -0,0 +1,34 @@
+import BigNumber from 'bignumber.js'
+import erc20 from 'src/abis/IERC20.json'
+import { tokenAmountInSmallestUnit } from 'src/tokens/saga'
+import Logger from 'src/utils/Logger'
+import { publicClient } from 'src/viem'
+import { getViemWallet } from 'src/web3/contracts'
+import networkConfig from 'src/web3/networkConfig'
+import { call } from 'typed-redux-saga'
+import { getAddress } from 'viem'
+
+// example usage
+export function* send(
```

Comment:
I think it was a really good idea to write this function to confirm that the changes work as expected and lend themselves to a good interface and devX for upcoming work. should it have a todo for plugging it into a main flow and a link to the GH issue where that work is going to get done? just so it doesn't get forgotten about and become dead code

Reply:
```suggestion
export function* send( // todo(ACT-786): build on this when implementing viem version of sendAndMonitorTransaction
```
maybe this would suffice